snc.sh

```diff
@@ -169,6 +169,31 @@ export OPENSHIFT_INSTALL_INVOKER="codeReadyContainers"
 
 ${OPENSHIFT_INSTALL} --dir $INSTALL_DIR create cluster --log-level debug
 
+# export the kubeconfig
+export KUBECONFIG=$INSTALL_DIR/auth/kubeconfig
+
+# Add kubelet-bootstrap-cred-manager daemonset
+oc apply -f kubelet-bootstrap-cred-manager-ds.yaml
+
+# Delete the current csr signer to get new request. 
+oc delete secrets/csr-signer-signer secrets/csr-signer -n openshift-kube-controller-manager-operator
+
+# Wait for 5 min to make sure cluster is stable again.
+sleep 300
+
+# Remove the 24 hours certs and bootstrap kubeconfig
+# This kubeconfig again get generated and new certs will be created in pki folder
```

Comment:
"this kubeconfig will be regenerated" instead of "This kubeconfig again get generated "?